src/openapi-generator.ts

```diff
@@ -1113,12 +1098,12 @@ export class OpenAPIGenerator {
 
   private applySchemaMetadata(
     initialData: SchemaObject | ParameterObject | ReferenceObject,
-    metadata: Partial<ZodOpenAPIMetadata>
+    metadata?: Partial<ZodOpenAPIMetadata>
```

Comment:
I am not a big fan of this even if it helps with some of the code.

Basically right now we have a method saying "apply metadata" that can work without metadata - that's very odd in my opinion. I guess we can work around this by renaming the method :thinking:. Not sure what could be a suitable name.

And one more thing - if we do end up renaming it and sticking with the optional, there are a couple more instances where a ternary operator was used to cover for undefined metadata.